src/FSharp.Markdown/HtmlFormatting.fs

```diff
@@ -67,19 +67,19 @@ let noBreak (ctx:FormattingContext) () = ()
 
 /// Write MarkdownSpan value to a TextWriter
 let rec formatSpan (ctx:FormattingContext) = function
-  | LatexDisplayMath(body) ->
+  | LatexDisplayMath(body, _) ->
       // use mathjax grammar, for detail, check: http://www.mathjax.org/
       ctx.Writer.Write("<span class=\"math\">\\[" + (htmlEncode body) + "\\]</span>")
-  | LatexInlineMath(body) ->
+  | LatexInlineMath(body, _) ->
       // use mathjax grammar, for detail, check: http://www.mathjax.org/
       ctx.Writer.Write("<span class=\"math\">\\(" + (htmlEncode body) + "\\)</span>")
 
-  | AnchorLink(id) -> ctx.Writer.Write("<a name=\"" + id + "\">&#160;</a>") 
-  | EmbedSpans(cmd) -> formatSpans ctx (cmd.Render())
-  | Literal(str) -> ctx.Writer.Write(str)
-  | HardLineBreak -> ctx.Writer.Write("<br />" + ctx.Newline)
-  | IndirectLink(body, _, LookupKey ctx.Links (link, title)) 
-  | DirectLink(body, (link, title)) -> 
+  | AnchorLink(id, _) -> ctx.Writer.Write("<a name=\"" + id + "\">&#160;</a>") 
+  | EmbedSpans(cmd, _) -> formatSpans ctx (cmd.Render())
+  | Literal(str, _) -> ctx.Writer.Write(str)
+  | HardLineBreak(_) -> ctx.Writer.Write("<br />" + ctx.Newline)
+  | IndirectLink(body, _, LookupKey ctx.Links (link, title), _) 
+  | DirectLink(body, (link, title), _) -> 
```

Comment:
We definitely do _not_ need to use the named syntax here - adding `_` is just fine here :). 

(I think it is important to have the named syntax in the docs, but not everywhere in the codebase.)